.github/workflows/ci-app-examples.yml

```diff
@@ -43,7 +43,7 @@ jobs:
 
       # TODO: use replace_oldest_ver() instead
       - name: Set min. dependencies
-        if: matrix.requires == 'oldest'
+        if: ${{ matrix.requires == 'oldest' }}
```

Comment:
Did this not work?

Reply:
it is official way and found some flakiness with similar contitions